AspNetCore.SassCompiler.Tasks/CompileSass.cs

```diff
@@ -220,5 +193,37 @@ private IEnumerable<ITaskItem> GenerateScopedCss(SassCompilerOptions options)
                 yield return generatedFile;
             }
         }
+
+        private (bool Success, string Output, string Error) GenerateCss(string arguments)
+        {
+            var compiler = new Process();
+            compiler.StartInfo = new ProcessStartInfo
+            {
+                FileName = Command,
+                Arguments = arguments,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+            };
+
+            string error = null;
+            compiler.ErrorDataReceived += (sender, e) =>
+            {
+                error += e.Data + "\r\n";
```

Comment:
I think we should use `Environment.NewLine` instead of `\r\n` so it works correctly on unix/mac as well